src/main/java/uk/gov/digital/ho/hocs/queue/ukvi/UKVIComplaintConsumer.java

```diff
@@ -61,13 +61,7 @@ public void configure() {
 
     @PreDestroy
```

Comment:
In the other projects I've moved the PreDestroy to the @SpringBootApplication class. Pleasse could you do the same.

Reply:
Moved to application entry point